src/tests/process/dhcpv6/server.conf

```diff
@@ -0,0 +1,112 @@
+#  -*- text -*-
+#
+#  test configuration file.  Do not install.
+#
+#  $Id$
+#
+
+#
+#  Minimal dhcpv6.conf for testing
+#
+
+testdir      = $ENV{TESTDIR}
+output       = $ENV{OUTPUT}
+run_dir      = ${output}
+raddb        = raddb
+pidfile      = ${run_dir}/radiusd.pid
+panic_action = "gdb -batch -x src/tests/panic.gdb %e %p > ${run_dir}/gdb.log 2>&1; cat ${run_dir}/gdb.log"
+
+maindir      = ${raddb}
+radacctdir   = ${run_dir}/radacct
+modconfdir   = ${maindir}/mods-config
+certdir      = ${maindir}/certs
+cadir        = ${maindir}/certs
+test_port    = $ENV{TEST_PORT}
+
+#  Only for testing!
+#  Setting this on a production system is a BAD IDEA.
+security {
+	allow_vulnerable_openssl = yes
+}
+
+policy {
+	$INCLUDE ../policy.conf
+	$INCLUDE ${raddb}/policy.d/control
+}
+
+#
+#  This virtual server is chosen for processing requests when using:
+#
+#	radiusd -Xd src/tests/ -i 127.0.0.1 -p 12340 -n test
+#
+server dhcpv6 {
+	namespace = dhcpv6
+
+	all_dhcp_servers_and_relays = FF02::1:2
+	interface = eth0
+	port = 547
+
+	listen local_network {
+		type = Request
+		type = Solicit
+		type = Information-Request
+
+		transport = udp
+
+		udp {
+			ipaddr = 2001:db8::
+			port = ${...port}
+			interface = ${...interface}
+		}
+	}
+
+	dhcpv6 {
+#		status_code_on_success = no
+#		send_failure_message = no
+#		move_failure_message_to_parent = yes
+	}
+
+	recv Solicit {
+		ok
+	}
+
+	send Advertise {
+		&reply += {
+			&IA-PD.IAID = 33752069
+			&IA-PD.T1 = 150
+			&IA-PD.T2 = 250
+
+			&IA-PD.Options.IA-PD-Prefix.Preferred-Lifetime = 250
+			&IA-PD.Options.IA-PD-Prefix.Valid-Lifetime = 300
+			&IA-PD.Options.IA-PD-Prefix.IPv6-Prefix = 2a00:1:1:100::/56
+
+			&Preference = 10
+
+			&DNS-Servers = 2a01:cafe:1
+			&AFTR-Name = "tapioca.net"
+		}
+
+		ok
+	}
+
+	recv Request {
+		ok
+	}
+
+	send Reply {
+		&reply += {
+			&IA-PD.IAID = 33752069
```

Comment:
Is the IAID not auto-populated?  We should definitely fix that.  Something like if there's a `IA-` container in the reply, and only one container of that type if the request, copy the IAID over automatically...